src/Compute/Compute/Models/PSVirtualMachineInstanceView.cs

```diff
@@ -22,6 +22,10 @@ public class PSVirtualMachineInstanceView
         public string ResourceGroupName { get; set; }
 
         public string Name { get; set; }
+        public string ComputerName { get; set; }
```

Comment:
Need a changelog entry for thsi change

Reply:
@markcowl  Added the changelog.  Thanks.